extide/o.apache.tools.ant.module/src-bridge/org/apache/tools/ant/module/bridge/impl/NbInputHandler.java

```diff
@@ -88,9 +88,9 @@ private JPanel createPanel(InputRequest request) {
         JLabel prompt = new JLabel();
         if (promptText.contains("\n")) { // #35712
             prompt.setText("<html>" + promptText.
-                    replaceAll("&", "&amp;").
-                    replaceAll("<", "&lt;").
-                    replaceAll("\r?\n", "<br>"));
```

Comment:
Just on this: changing `replaceAll("\r?\n", "<br>")` to `replace("\r?\n", "<br>")` is not equivalent, right?

Reply:
You are correct and apologies for the mistake. This mistake will be corrected and a more comprehensive replaceAll() to replace() will be added to this pull request in the upcoming days.